client/src/main/java/com/influxdb/client/internal/TasksApiImpl.java

```diff
@@ -120,13 +122,25 @@ public Task findTaskByID(@Nonnull final String taskID) {
         return findTasks(null, null, orgID);
     }
 
+
     @Nonnull
     @Override
     public List<Task> findTasks(@Nullable final String afterID,
                                 @Nullable final String userID,
                                 @Nullable final String orgID) {
+        final TaskQuery taskQuery = new TaskQuery();
+        taskQuery.setAfter(afterID);
+        taskQuery.setUser(userID);
+        taskQuery.setOrgID(orgID);
+
+        return findTasks(taskQuery);
+    }
 
-        Call<Tasks> call = service.getTasks(null, null, afterID, userID, null, orgID, null, null);
+    @NotNull
```

Comment:
Use `javax.annotation.Nonnull` anotation
```suggestion
    @Nonnull
```